### ISSUE TITLE
[BUG] eww systray draws its own menus rather than letting the applications draw their own menus

### ISSUE THREAD

**Issue by victorz** [elkowar/eww#1081]
### Checklist before submitting an issue

- [X] I have searched through the existing [closed and open issues](https://github.com/elkowar/eww/issues?q=is%3Aissue) for eww and made sure this is not a duplicate
- [X] I have specifically verified that this bug is not a common [user error](https://github.com/elkowar/eww/issues?q=is%3Aissue+label%3Ano-actual-bug+is%3Aclosed)
- [X] I am providing as much relevant information as I am able to in this bug report (Minimal config to reproduce the issue for example, if applicable)

### Description of the bug

The systray widget seems to draw its own menus, whereas I would like it to let the program draw its own menus instead, if that makes sense. I'm not sure how it all works, but I have a couple of video clips I can show to demonstrate.

---

_Apologies if this should be a feature request instead(?). Let me know if so and I'll do my best to convert it to one._

### Reproducing the issue

Use the systray widget somewhere, and right-click an icon to bring up their menu. Preferably some program that renders a custom menu, e.g. Telegram Desktop.

Here's a video clip to show what it looks like with eww's systray widget.

https://github.com/elkowar/eww/assets/7494/6cb12048-0701-45c2-ba12-aa0e2d144aa4

### Expected behaviour

Here's a video of `stalonetray`, and I am right-clicking the Telegram icon to bring up its menu -- seemingly a custom menu rendered by Telegram itself, rather than stalonetray.

https://github.com/elkowar/eww/assets/7494/1ae0c8f1-d218-4ecb-aece-aa0e0992d793

### Additional context

Anyway, I'm super stoked to have this functionality in eww! This is just great! :slightly_smiling_face: :clap:

**Comment by cry0x11** [comment]
> ### Checklist before submitting an issue
> * [x]  I have searched through the existing [closed and open issues](https://github.com/elkowar/eww/issues?q=is%3Aissue) for eww and made sure this is not a duplicate
> * [x]  I have specifically verified that this bug is not a common [user error](https://github.com/elkowar/eww/issues?q=is%3Aissue+label%3Ano-actual-bug+is%3Aclosed)
> * [x]  I am providing as much relevant information as I am able to in this bug report (Minimal config to reproduce the issue for example, if applicable)
> 
> ### Description of the bug
> The systray widget seems to draw its own menus, whereas I would like it to let the program draw its own menus instead, if that makes sense. I'm not sure how it all works, but I have a couple of video clips I can show to demonstrate.
> 
> _Apologies if this should be a feature request instead(?). Let me know if so and I'll do my best to convert it to one._
> 
> ### Reproducing the issue
> Use the systray widget somewhere, and right-click an icon to bring up their menu. Preferably some program that renders a custom menu, e.g. Telegram Desktop.
> 
> Here's a video clip to show what it looks like with eww's systray widget.
> 
>  eww_systray.mp4 
> ### Expected behaviour
> Here's a video of `stalonetray`, and I am right-clicking the Telegram icon to bring up its menu -- seemingly a custom menu rendered by Telegram itself, rather than stalonetray.
> 
>  stalonetray.mp4 
> ### Additional context
> Anyway, I'm super stoked to have this functionality in eww! This is just great! 🙂 👏

Have you tried to remove ```all: unset;``` at the top of the default SCSS file?
```
* {
    all:unset;
}
```
Removing it will revert back to the default GTK style from your theme (unless you already styled the systray menus). It's not exactly what you are asking for, but better than nothing.

If you wish to style it yourself, I wrote about my attempt to style it with radio buttons, check boxes and sub-menu arrows visible, please refer to my comment here https://github.com/elkowar/eww/issues/1067#issuecomment-2078227549

**Comment by victorz** [comment]
> Have you tried to remove ```all: unset;``` at the top of the default SCSS file?
> ```
> * {
>     all:unset;
> }
> ```
> Removing it will revert back to the default GTK style from your theme (unless you already styled the systray menus). It's not exactly what you are asking for, but better than nothing.
> 
> If you wish to style it yourself, I wrote about my attempt to style it with radio buttons, check boxes and sub-menu arrows visible, please refer to my comment here https://github.com/elkowar/eww/issues/1067#issuecomment-2078227549

Yes, I've tried without that unset. (I never had one to begin with.)

I don't think the Telegram menu is GTK though? It's a custom menu drawn by Telegram, surely? Did you take a look at the video clips?
